.github/workflows/release.yml

```
@@ -0,0 +1,194 @@
+            All release artifacts are signed with [cosign](https://github.com/sigstore/cosign).
+
+            To verify a release artifact:
+            ```bash
+            cosign verify-blob \
+              --certificate-identity=https://github.com/${{ github.repository }}/.github/workflows/release.yml@${{ github.ref }} \
+              --certificate-oidc-issuer=https://token.actions.githubusercontent.com \
+              --signature=cloudpull-checksums.txt.sig \
+              cloudpull-checksums.txt
+            ```
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)